src/components/reviews.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import { fetchReviews } from "../api";
+
+const Reviews = () => {
+    const [reviewsData, setReviews] = useState([]);
+
+    useEffect(() => {
+        fetchReviews().then((data) => {
+            setReviews(data);
+        });
+    });
```

Comment:
Currently your useEffect() function is missing the dependency array, causing an infinite re-render and an infinite fetch loop. Please include it. ([]).